developers/weaviate/starter-guides/managing-collections/collections-scaling-limits.mdx

```diff
@@ -0,0 +1,173 @@
+---
+title: Do you have too many collections?
+sidebar_label: Scaling limits with collections
+sidebar_position: 1
+image: og/docs/more-resources.jpg
+# tags: ['performance']
+---
+
+:::info
+Added in `v1.25.34`, `v1.26.18`, `1.27.15`, `1.28.9`, `1.29.2`
+:::
+
+:::caution You hit the collections count limit?
+If you see an error that the collection count limit has been reached, it means you **can't create any more collections** because the number of collections is affecting performance. This limit exists to control indexing, storage, and management complexity.<br /><br />
+**Instead of simply raising the limit, consider rethinking your architecture**. If you really need to change the limit, use the [`MAXIMUM_ALLOWED_COLLECTIONS_COUNT`](/developers/weaviate/config-refs/env-vars) environment variable.
+:::
+
+This guide offers an overview of the available architectural choices of using **multi-tenancy** or defining a **dedicated collection** for each subset of data.
+
+Consider a scenario where you are developing a SaaS platform for product recommendations, allowing users (merchants) to import and vectorize their product data. The users of this platform require the ability to efficiently perform vector searches to generate personalized recommendations for users on their respective e-commerce platforms.
```

Comment:
Hey couple of thoughts about this:

- From an end user perspective, they might not care about "import and vectorize" data or "efficiently perform vector searches" - imo that is the platform builder's task.
- The point we want to highlight is that each user will only ever deal with their own data. 

So - wdyt about:

> Consider a scenario where a builder is developing a SaaS platform for product recommendations, allowing users (merchants) to recommend products to their shoppers. In this scenario, each merchant will upload and work with their own data only. 

One option for the builder is to create...